std/numeric.d

```diff
@@ -2572,27 +2572,55 @@ GapWeightedSimilarityIncremental!(R, F) gapWeightedSimilarityIncremental(R, F)
 
 /**
 Computes the greatest common divisor of $(D a) and $(D b) by using
-Euclid's algorithm.
+Euclid's or Stein's algorithm depending on compiler.
```

Comment:
> Oh, you may also add Web links to the algorithms:

@Darredevil FYI there's also the `LINK2` macro which allows to have clickable/copy-pasteable links in the source code. For more information, see e.g. this [discussion](https://github.com/dlang/phobos/pull/4411#issuecomment-226392170).